README.md

```diff
@@ -3,7 +3,11 @@
 [![Crowdin Badge](https://d322cqt584bo4o.cloudfront.net/phoenicis-scripts/localized.svg)](https://crowdin.com/project/phoenicis-scripts)
 
 # How to add a script
-1. Select the right category
+1. Clone this repository
+
+2. Add the local checkout as local repository in Phoenicis (see [instructions](https://phoenicisorg.github.io/phoenicis/Users/repository/#local-directory))
+
+3. Select the right category
```

Comment:
How do you select the category?

Reply:
Well, where does the script belong to.

Reply:
Yes I know that, but normally when I read or hear `select` I think about clicking on it, which is not what you mean here.
I think you want to say something like:
```
3. Select the right category through the use of its path in the next steps
```

Reply:
I get your point but I don't think this makes it clearer.